src/components/projects/project-edit-form/project-edit-container/project-edit-container.tsx

```diff
@@ -41,10 +53,28 @@ const ProjectEditContainer: React.FC<EditFormContainerProps> = ({ wbsNum, proj,
   const updateBom = (url: string | undefined) => setBom(url);
   const updateGDrive = (url: string | undefined) => setGDrive(url);
 
-  const [goals, setGoals] = useState(proj!.goals.map((goal) => goal.detail));
-  const [features, setFeatures] = useState(proj!.features.map((feature) => feature.detail));
-  const [otherConstraints, setOther] = useState(
-    proj!.otherConstraints.map((constraint) => constraint.detail)
+  const { mutateAsync } = useEditSingleProject();
+
+  const [goals, setGoals] = useState<{ id?: number; detail: string }[]>(
+    proj!.goals
+      .filter((goal) => !goal.dateDeleted)
```

Comment:
I guess this should fix the issue, i think i might have submitted a bug ticket for it? I like it